src/sidebar/components/PendingUpdatesButton.tsx

```diff
@@ -19,11 +20,18 @@ function PendingUpdatesButton({
   const store = useSidebarStore();
   const pendingUpdateCount = store.pendingUpdateCount();
   const hasPendingUpdates = store.hasPendingUpdates();
+  const applyPendingUpdates = () => streamer.applyPendingUpdates();
+
+  useShortcut('l', () => hasPendingUpdates && applyPendingUpdates());
```

Comment:
I checked what was the shortcut used by some social networks to load new available updates: Twitter uses the dot, and I couldn't find relevant information for other social networks.

Lyza expressed some legit concerns around using the dot, in a [previous draft PR](https://github.com/hypothesis/client/pull/5307#discussion_r1131135905), so I have set the `l` after "load", but I'm open to suggestions.

Reply:
I don't know the history behind the "." shortcut, but according to https://hyp.is/GO2gksPmEe2YyovzGjL3cw/help.twitter.com/en/using-twitter/how-to-tweet, "l" and "r" are taken for "like" and "reply", so aren't available for load / refresh.